lib/jekyll/renderer.rb

```diff
@@ -265,5 +268,10 @@ def output_exts
         c.output_ext(document.extname)
       end.compact
     end
+
+    private
+    def liquid_options
+      @liquid_options ||= site.config["liquid"]
```

Comment:
These could be in `lib/jekyll/liquid_renderer/file.rb` and be merged into the `info` hash in that class to isolate liquid concerns from here to there? What do you think?

Reply:
Both `LiquidRenderer` and `LiquidRenderer::File` have no `info` hash.. Are you suggesting, I add it..?

Reply:
merge could slow down rendering especially when each call to `renderer.run` is going to call a `merge`.. so deferring that for the future..